OKTA-508890 Update the android browser redirect guide to okta-mobile-kotlin. (#3464)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md

> 1. Create a new Kotlin Android project with an **Empty Activity** using Android Studio (we called it `browser_sign_in`). For information on creating a project, see the [Android Studio documentation](https://developer.android.com/training/basics/firstapp/creating-project).

Author: This is mostly just a carry over from the previous guide. I like linking out to the android developer docs. I've gotten a lot of questions about how to do these things from people prototyping Android, that aren't Android experts. I don't have a strong feeling though here, so happy to remove it if you do.

> 2. The SDK uses features not available in all Android versions, add the desugaring SDK.

Author: The suggested text wasn't quite accurate, the next commit will contain an update (but please double check me again).

```gradle
dependencies {
    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
}
```

> > **Note**: If you're using the Okta CLI, you can also run `okta start android-kotlin` to create an app. This command creates an OIDC app in Okta, downloads the [okta-android-kotlin-sample](https://github.com/okta-samples/okta-android-kotlin-sample), and configures it to work with the OIDC app. This quickstart uses a basic Android project instead, as it's easier to understand the Okta-specific additions if you work through them yourself.

Reviewer: I'm not sure if this note is really needed, though if it is...

```suggestion
> **Note**: The Okta CLI can create a Kotlin app for Android that's pre-configured for an Okta org by using the command `okta start android-kotlin`. This QuickStart uses a basic project to show you how to configure an app.
```
